app/_data/tables/version_errors_konnect.yml

```diff
@@ -1856,23 +2059,282 @@ messages:
   - ID: P189
     Severity: error
     Description: |
-      Only Kong Gateway versions >= 3.1.0.0 and < 3.1.1.2 support the wRPC protocol which 
-      has been removed from Kong Gateway >= 3.2.0.0. Konnect will be removing support for 
-      these releases of Kong Gateway on 2023/09/30. After this date these versions of 
-      Kong Gateway will not be able to establish connections with Konnect to retrieve 
-      configuration changes. Kong recommends upgrading to Kong Gateway 3.4.0.0 LTS.
+      Kong Gateway versions >= 3.1.0.0 and < 3.1.1.2 only support the wRPC protocol
+      which has been removed from Kong Gateway >= 3.2.0.0. Konnect will be removing
+      support for these releases of Kong Gateway on 2023/09/30. After this date these
+      versions of Kong Gateway will no longer be able to establish connections with
+      Konnect and will not receive configuration changes. You may upgrade to Kong
+      Gateway >= 3.1.1.2; however Kong recommends upgrading to Kong Gateway 3.4.0.0
+      LTS.
     Resolution: |
       Please upgrade Kong Gateway to version `3.4.0.0` or above.
     DocumentationURL: |
       N/A
   - ID: P190
     Severity: warning
     Description: |
-      For the `vaults` entity, the `prefix` value contains hyphens. This is not
+      For the `vaults` entity, the'prefix` value contains hyphens. This is not
```

Comment:
here too

Reply:
This is because a utility function that converts single quotes into backticks purposefully doesn't convert single quotes that are immediately between two alphabetical characters (this is a fancy way of saying apostrophes). In the cases here, a space character is missing from the multiline string literal, and the first or last word in the line just so happens to be wrapped in single quotes. 

Writing it out is sort of overkill, it will make more sense in the fixup PR over in KOKO. I'll tag you in it!